pkgs/by-name/st/stash/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchurl, }:
+
+stdenv.mkDerivation rec {
+  pname = "stash";
+  version = "0.25.1";
+
+  src = fetchurl {
+    url = "https://github.com/stashapp/stash/releases/download/v${version}/stash-linux";
+    sha256 = "sha256-Rb4x6iKx6T9NPuWWDbNaz+35XPzLqZzSm0psv+k2Gw4=";
+  };
+
+  unpackPhase = "true";
```

Comment:
Not needed

Reply:
It failed because the unpack phase is not needed. I added the `dontUnpack` attribute now and set it to `true` since there is no unpacking needed. Is that a better approach or do you suggest something else?